From: Masanari Iida <standby24x7@gmail.com>
Date: Tue, 22 Jul 2014 23:42:37 +0900
Subject: [241/249] staging: rtl8723au: Fix typo in rtw_xmit.c
Origin: https://git.kernel.org/linus/c582db4cb82e69f92ca5111b5043129438d5fae9

This patch fix spelling typo in comments within rtw_xmit.c

Signed-off-by: Masanari Iida <standby24x7@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723au/core/rtw_xmit.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

Index: linux-3.16.7/drivers/staging/rtl8723au/core/rtw_xmit.c
===================================================================
--- linux-3.16.7.orig/drivers/staging/rtl8723au/core/rtw_xmit.c
+++ linux-3.16.7/drivers/staging/rtl8723au/core/rtw_xmit.c
@@ -1197,7 +1197,7 @@ int rtw_xmitframe_coalesce23a(struct rtw
 			mpdu_len -= pattrib->icv_len;
 
 		if (bmcst)
-			/*  don't do fragment to broadcat/multicast packets */
+			/*  don't do fragment to broadcast/multicast packets */
 			mem_sz = min_t(s32, data_len, pattrib->pktlen);
 		else
 			mem_sz = min_t(s32, data_len, mpdu_len);
@@ -2011,7 +2011,7 @@ int xmitframe_enqueue_for_sleeping_sta23
 
 			/* DBG_8723A("enqueue, sq_len =%d, tim =%x\n", psta->sleepq_len, pstapriv->tim_bitmap); */
 
-			/* tx bc/mc packets after upate bcn */
+			/* tx bc/mc packets after update bcn */
 			update_beacon23a(padapter, WLAN_EID_TIM, NULL, false);
 
 			/* spin_unlock_bh(&psta->sleep_q.lock); */
@@ -2068,7 +2068,7 @@ int xmitframe_enqueue_for_sleeping_sta23
 				pstapriv->tim_bitmap |= CHKBIT(psta->aid);
 
 				if (psta->sleepq_len == 1) {
-					/* upate BCN for TIM IE */
+					/* update BCN for TIM IE */
 					update_beacon23a(padapter, WLAN_EID_TIM,
 							 NULL, false);
 				}
@@ -2227,7 +2227,7 @@ void wakeup_sta_to_xmit23a(struct rtw_ad
 	if (psta->sleepq_len == 0) {
 		pstapriv->tim_bitmap &= ~CHKBIT(psta->aid);
 
-		/* upate BCN for TIM IE */
+		/* update BCN for TIM IE */
 		update_mask = BIT(0);
 
 		if (psta->state&WIFI_SLEEP_STATE)
@@ -2274,7 +2274,7 @@ void wakeup_sta_to_xmit23a(struct rtw_ad
 			pstapriv->tim_bitmap &= ~BIT(0);
 			pstapriv->sta_dz_bitmap &= ~BIT(0);
 
-			/* upate BCN for TIM IE */
+			/* update BCN for TIM IE */
 			/* update_BCNTIM(padapter); */
 			update_mask |= BIT(1);
 		}
@@ -2348,7 +2348,7 @@ void xmit_delivery_enabled_frames23a(str
 		    (wmmps_ac)) {
 			pstapriv->tim_bitmap &= ~CHKBIT(psta->aid);
 
-			/* upate BCN for TIM IE */
+			/* update BCN for TIM IE */
 			update_beacon23a(padapter, WLAN_EID_TIM, NULL, false);
 		}
 	}
